elasticsearch/src/main/java/org/apache/zeppelin/elasticsearch/ElasticsearchInterpreter.java

```diff
@@ -0,0 +1,369 @@
+          .build());
+  }
+
+  private final Gson gson = new GsonBuilder().setPrettyPrinting().create();
+  private Client client;
+  private String host = "localhost";
+  private int port = 9300;
+  private String clusterName = "elasticsearch";
+
+  public ElasticsearchInterpreter(Properties property) {
+    super(property);
+    this.host = getProperty(ELASTICSEARCH_HOST);
+    this.port = Integer.parseInt(getProperty(ELASTICSEARCH_PORT));
+    this.clusterName = getProperty(ELASTICSEARCH_CLUSTER_NAME);
+  }
+
+  @Override
+  public void open() {
+    try {
+      final Settings settings = Settings.settingsBuilder()
+        .put("cluster.name", clusterName).build();
+      client = TransportClient.builder().settings(settings).build()
+        .addTransportAddress(new InetSocketTransportAddress(InetAddress.getByName(host), port));
+    }
+    catch (IOException e) {
+      logger.error("Open connection with Elasticsearch", e);
+    }
+  }
+
+  @Override
+  public void close() {
+    if (client != null) {
+      client.close();
+    }
+  }
+
+  @Override
+  public InterpreterResult interpret(String cmd, InterpreterContext interpreterContext) {
+    logger.info("Run Elasticsearch command '" + cmd + "'");
+    // RULES for an elasticsearch command:
+    // <Method> /<index>/<type>/ <option> <JSON>
+    // Method = search, get, index, delete, count
+
+    final String[] items = StringUtils.split(cmd.trim(), " ", 3);
+
+    if (items.length < 2) {
+      return new InterpreterResult(InterpreterResult.Code.ERROR, "Arguments missing");
+    }
+
+    final String method = items[0];
+    final String url = items[1];
+    final String data = items.length > 2 ? items[2].trim() : null;
+
+    final String[] urlItems = StringUtils.split(url.trim(), "/");
+
+    try {
+      if ("get".equalsIgnoreCase(method)) {
+        return processGet(urlItems);
+      }
+      else if ("count".equalsIgnoreCase(method)) {
+        return processCount(urlItems);
+      }
+      else if ("search".equalsIgnoreCase(method)) {
+        return processSearch(urlItems, data);
+      }
+      else if ("index".equalsIgnoreCase(method)) {
+        return processIndex(urlItems, data);
+      }
+      else if ("delete".equalsIgnoreCase(method)) {
+        return processDelete(urlItems);
+      }
+    
+      return new InterpreterResult(InterpreterResult.Code.ERROR, "Unknown method");
+    }
+    catch (Exception e) {
+      return new InterpreterResult(InterpreterResult.Code.ERROR, "Error : " + e.getMessage());
+    }
+  }
+
+  @Override
+  public void cancel(InterpreterContext interpreterContext) {
+    // Nothing to do
+  }
+
+  @Override
+  public FormType getFormType() {
+    return FormType.SIMPLE;
+  }
+
+  @Override
+  public int getProgress(InterpreterContext interpreterContext) {
+    return 0;
+  }
+
+  @Override
+  public List<String> completion(String s, int i) {
+    if (i == 0) {
+      return Arrays.asList("search", "get", "index", "delete", "count");
```

Comment:
it looks like you have taken out completion? I think it's still worthwhile to have some of it..